docs/architecture/overview.md

```diff
@@ -0,0 +1,193 @@
+
+DCP performs the following tasks:
+
+- Prepares the resources for execution:
+  - Configures service endpoints.
+    - Assigns names and ports dynamically, unless explicitly set (DCP ensures that the ports are available and not in use by other processes).
+  - Initializes container networks.
+  - Pulls container images based on their applied [pull policy](xref:Aspire.Hosting.ApplicationModel.ImagePullPolicy).
+  - Creates and starts containers.
+  - Runs executables with the required arguments and environment variables.
+- Monitors resources:
+  - Provides change notifications about objects managed within DCP, including process IDs, running status, and exit codes (the app host subscribes to these changes to manage the [application's lifecycle](../app-host/eventing.md) effectively).
+- Starts the developer dashboard.
+
+Continuing from the [diagram in the previous](#app-host-dcp-flow) section, consider the following diagram that helps to visualize the responsibilities of DCP:
+
+<span id="dcp-architecture"></span>
+
+:::image type="content" source="media/dcp-architecture-thumb.png" alt-text="A diagram depicting the architecture of the Developer Control Plane (DCP)." lightbox="media/dcp-architecture.png":::
+
+DCP logs are streamed back to the app host, and then the app host pushes them to the developer dashboard. DCP registers default commands for the developer dashboard as well, such as start, stop, and restart. For more information on commands, see [Custom resource commands in .NET Aspire](../fundamentals/custom-resource-commands.md).
+
+## Developer dashboard
+
+The [.NET Aspire developer dashboard](../fundamentals/dashboard/overview.md) is a powerful tool designed to simplify local development and resource management. It also supports a [standalone mode](../fundamentals/dashboard/standalone.md) and integrates seamlessly when publishing to Azure Container Apps. With its intuitive interface, the dashboard empowers developers to monitor, manage, and interact with application resources effortlessly.
+
+### Monitor and manage resources
+
+The dashboard provides a user-friendly interface for inspecting resource states, viewing logs, and executing commands. Whether you're debugging locally or deploying to the cloud, the dashboard ensures you have full visibility into your application's behavior.
+
+### Built-in and custom commands
+
+The dashboard relies on DCP to expose a set of default commands for resources, such as start, stop, and restart. These commands are readily available in the dashboard UI, streamlining resource management. For more information, see [Stop or Start a resource](../fundamentals/dashboard/explore.md#stop-or-start-a-resource).
```

Comment:
See my comment above regarding commands being an Aspire app model concept and not DCP concept.